app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -77,6 +77,22 @@ def ping
       render json: { status: 'OK' }
     end
 
+    api :PUT, "/organizations/:organization_id/enable_simple_content_access",
```

Comment:
I think these endpoints should be organized like this `/organization/:id/simple_content_access/enable` rather then `enable_simple_content_access` - it's more consistent with our other APIs